Pyjion/pyjit.cpp

```diff
@@ -75,20 +108,43 @@ extern "C" __declspec(dllexport) PyJittedCode* JitCompile(PyCodeObject* code) {
 #ifdef DEBUG_TRACE
         printf("Compilation failure #%d\r\n", ++failCount);
 #endif
-        return nullptr;
-    }
-
-    auto jittedCode = (PyJittedCode*)PyJittedCode_New();
-    if (jittedCode == nullptr) {
-        // OOM
-        delete res;
-        return nullptr;
+        jittedCode->j_failed = true;
+        return false;
     }
 
-    g_jittedCode[jittedCode] = res;
+    g_pyjionJittedCode[jittedCode] = res;
     jittedCode->j_evalfunc = &Jit_EvalHelper;
     jittedCode->j_evalstate = res->get_code_addr();
-    return jittedCode;
+    return true;
+}
+
+static PY_UINT64_T HOT_CODE = 20000;
+
+extern "C" __declspec(dllexport) PyObject *EvalFrame(PyFrameObject *f, int throwflag) {
```

Comment:
I'm a little curious if we want to flow throwflag through here or not... It seems like this is basically an internal Python implementation and we just never want to pass the flag through (e.g. not call the external handler when it's set, and then assume it's always 0 when executing).